fairlearn/reductions/grid_search/grid_search.py

```diff
@@ -181,7 +194,7 @@ def fit(self, X, y, aux_data=None, **kwargs):
                     y_reduction = 1 * (weights > 0)
                     weights = weights.abs()
                 else:
-                    y_reduction = y
+                    y_reduction = y_vector
```

Comment:
This was an actual bug which my tests found :-)

Reply:
I noticed! nice!